src/java/io/bazel/rulesscala/jar/BUILD

```diff
@@ -0,0 +1,5 @@
+java_binary(name = "jar",
```

Comment:
FWIW, @bazel_tools//tools/zip:zipper is a zip implementation that strip out timestamp. Might be easier to refer to it that to fork SingleJar's Jar implementation. (Also faster)